fetch-doi-testing/src/main/java/no/sikt/nva/testing/http/WiremockHttpClient.java

```diff
@@ -0,0 +1,87 @@
+package no.sikt.nva.testing.http;
+
+import java.net.Socket;
+import java.net.http.HttpClient;
+import java.security.KeyManagementException;
+import java.security.NoSuchAlgorithmException;
+import java.security.cert.CertificateException;
+import java.security.cert.X509Certificate;
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.SSLEngine;
+import javax.net.ssl.X509ExtendedTrustManager;
+import org.junit.jupiter.api.Assertions;
+
+public final class WiremockHttpClient {
```

Comment:
An HttpsClient that trusts everything. Copied from doi-registrar.

Reply:
This is a far more complex approach compared to the few lines of mocking the client and its responses in the method where needed.

Reply:
True, however this needs to be done once, and we do not need to mock the HttpClient and its responses. It will pay off when we will test for other cases. In addition, we get to test that we are contacting the correct URIs for free and we get more detailed error responses.